cmd/app/link_test.go

```diff
@@ -537,6 +538,159 @@ func Test_Apps_Link(t *testing.T) {
 				require.Len(t, apps, 0)
 			},
 		},
+		"manifest source prompt should not display for Run-on-Slack apps with local manifest source": {
```

Comment:
note: New test case to assert that the manifest update prompt is NOT displayed for ROSI apps with a local manifest. I don't check for remote manifests, because that's handled in an existing test-case.